beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix.go

```diff
@@ -80,9 +80,9 @@ func setExecutionData(ctx context.Context, blk interfaces.SignedBeaconBlock, loc
 		}
 
 		// Use builder payload if the following in true:
-		// builder_bid_value * 100 > local_block_value * (local-block-value-boost + 100)
+		// builder_bid_value * builderBoostFactor(default 100) > local_block_value * (local-block-value-boost + 100)
```

Comment:
Hmmm I don't like this. The point of `builderBoostFactor` is that it replaces our current parameter `local-block-value-boost`   not to use both... I'm not sure what the right approach is here but at the very least we should log if there's a request and the local boost is set to non-zero

Reply:
I added a log for now we'll see if others have an opinion